KCAA/Services/TelegramApi/TelegramUpdateHandlers/TelegramCallbackQueryHandler.cs

```diff
@@ -359,6 +393,30 @@ private async Task SendKillCharacterKeyboard(long chatId, Lobby lobby, Player pl
             await _playerProvider.UpdatePlayer(player.Id, p => p.TelegramMetadata, player.TelegramMetadata);
         }
 
+        private async Task SendStealCharacterKeyboard(long chatId, Lobby lobby, Player player, string characterName, string gameAction)
```

Comment:
Method looks almost the same as SendKillCharacterKeyboard, maybe we should combine them?